lib/package.gi

```diff
@@ -246,6 +246,30 @@ BindGlobal( "AddPackageInfos", function( files, pkgdir, ignore )
             elif IsRecord( record.PackageDoc ) then
               record.PackageDoc:= [ record.PackageDoc ];
             fi;
+
+            #Normalize the format of 'Date', i.e. if it is the format yyyy-mm-dd
```

Comment:
Could you please put a space between the `#` character and the words in this PR, including in this line? I find it hard to read like this.